tests/pytests/test.py

```diff
@@ -2436,6 +2436,16 @@ def testAlias(env):
     env.cmd('ft.create', 'idx2', 'ON', 'HASH', 'PREFIX', 1, 'doc2', 'schema', 't1', 'text')
 
     env.expect('ft.aliasAdd', 'myIndex').error()
+
+    pre_indices = env.cmd('ft._list')
+    env.cmd('ft.create', 'tmp_index_name', 'ON', 'HASH', 'PREFIX', 1, 'doc2', 'schema', 't1', 'text')
+    env.expect('ft.aliasAdd', 'tmp_index_name', 'idx').error()
+    env.cmd('ft.drop', 'tmp_index_name')
+    post_indices = env.cmd('ft._list')
+    env.assertEqual(set(pre_indices), set(post_indices))
+    env.cmd('ft.aliasAdd', 'tmp_index_name', 'idx')
+    env.cmd('ft.aliasDel', 'tmp_index_name')
+
```

Comment:
Move to a separate test (will also solve "conflict" issues with existing indexes)